memo.js

```diff
@@ -0,0 +1,30 @@
+NO_VALUE = Symbol("NO_VALUE");
+
+class Trie {
```

Comment:
`Trie`?

Reply:
[Trie!](https://xlinux.nist.gov/dads/HTML/trie.html)

Reply:
I like how 50% of that article was about how to pronounce it.